recipes/pgrc/build.sh

```
@@ -0,0 +1,14 @@
+#! /bin/bash
+
+mkdir -p $PREFIX/bin
+
+mkdir build
+cd build
```

Shellcheck

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)